### ISSUE TITLE
django.core.exceptions.FieldError for Page model using base_form_class with custom form

### ISSUE THREAD

**Issue by bplociennik** [infoportugal/wagtail-modeltranslation#288]
I'm getting errors when I'm using Page model with custom `base_form_class` and field is not included in the model. I don't have it in translation.py of course.

**Error:**
`django_1      | django.core.exceptions.FieldError: Unknown field(s) (next_summer_label, current_winter_label, current_summer_label, next_winter_label) specified for PropertyPage`

**Model and form:**
```
class PropertyForm(BaseAdminPageForm):
    current_winter_label = forms.CharField()
    next_winter_label = forms.CharField()
    current_summer_label = forms.CharField()
    next_summer_label = forms.CharField()

class PropertyPage(BasePage):
    (...)
    base_form_class = PropertyForm

    pricing_related_panel = [
        MultiFieldPanel(
            [
                FieldPanel("current_winter_label"),
              ],
              heading=_("current winter season"),
             ),
          ]
     
     edit_handler = TabbedInterface(
        [
            (...)
            ObjectList(pricing_related_panel, heading=_("pricing related")),
        ]
    )
```

Without wagtail modeltranslation is working correctly.

Do you have any ideas on how to solve this issue?

**Comment by DiogoMarques29** [comment]
Can you show a stacktrace of the error?

Cheers

**Comment by DiogoMarques29** [comment]
A fix has been made in v0.10.14
